### ISSUE TITLE
Quick Fix form does not mark required fields

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#166]
Based on our partner's feedback:

> Mandatory fields (Alternative text, Position Combo box, Header Level combo box) info on Accessibility Checker info not conveyed for Keyboard or visual users. JAWS reads them as mandatory but the same info should be given for Keyboard only users. Please add \* next to fields and at bottom of dialog indicate what \* means ( ex: \* Required). This is violation of AVT Checkpoint 1.3d Forms.
